json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -271,17 +271,24 @@ protected function result() {
 		$request_body ['buffer_id'] = preg_replace( '/[^A-Za-z0-9]/', '', $request_body['buffer_id'] );
 		$request_body['item_ids'] = array_filter( array_map( array( 'Jetpack_JSON_API_Sync_Close_Endpoint', 'sanitize_item_ids' ), $request_body['item_ids'] ) );
 
-		$buffer = new Queue_Buffer( $request_body['buffer_id'], $request_body['item_ids'] );
 		$queue = new Queue( $queue_name );
 
+		$items = $queue->peek_by_id( $request_body['item_ids'] );
+
+		/** This action is documented in Full_Sync.php */
```

Comment:
This is outside of the sync package, so maybe let's provide a full Jetpack-root-relative path to the file.
```suggestion
		/** This action is documented in packages/sync/src/modules/Full_Sync.php */
```